docs: update documentation on new prometheus metrics (#14)
chartmuseum/www: content/docs/prometheus-metrics/_index.md

draft: false
---

ChartMuseum exposes its [Prometheus metrics](https://prometheus.io/docs/concepts/metric_types/) at the `/metrics` route on the main port. This can be enabled with the `--enable-metrics` command-line flag or the `ENABLE_METRICS` environment variable.

> > Note that the Kubernetes chart currently disables metrics by default (`ENABLE_METRICS=false` is set in the chart). The `--disable-metrics` command-line flag has be deprecated and will only be available in `v0.14.0` and prior.

Reviewer: We should mention that it’s disabled by default in general and not only disabled in the chart.

Author: okay will update it now

> > Note that the metric is disabled by default and on the Kubernetes chart (`ENABLE_METRICS=false` is set in the chart). The `--disable-metrics` command-line flag has be deprecated and will only be available in `v0.14.0` and prior.

Reviewer: ```suggestion
> Note that metrics are disabled by default (this includes the Kubernetes Helm chart). The `--disable-metrics` command-line flag has been deprecated and will only be available in `v0.14.0` and prior.
```
